### PR TITLE
[ADF-4492]Add claim and release task tests

### PR DESCRIPTION
**Pull request by cristinaj**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-4492

### CODE REVIEW

File: e2e/process-services-cloud/task-form-cloud-component.e2e.ts [hunk 1]
```diff
@@ -67,85 +70,268 @@ describe('Complete task - cloud directive', () => {
...
+
+            tasksCloudDemoPage.taskListCloudComponent().checkContentIsDisplayedById(toClaimProcessTask.list.entries[0].entry.id);
+            tasksCloudDemoPage.taskListCloudComponent().selectRowById(toClaimProcessTask.list.entries[0].entry.id);
+            taskDetailsCloudDemoPage.checkTaskDetailsHeaderIsDisplayed();
+
+            expect(taskDetailsCloudDemoPage.taskHeaderCloud().getStatus()).toEqual('ASSIGNED');
+            expect(taskDetailsCloudDemoPage.taskHeaderCloud().getAssignee()).toEqual('admin.adf');
+        });
+
+        // ADF-4315
```
Comment (davidcanonieto) [inline e2e/process-services-cloud/task-form-cloud-component.e2e.ts]: Does it make sense to keep the Jira ticket number?

File: lib/testing/src/lib/process-services-cloud/pages/task-list-cloud-component.page.ts [hunk 2]
```diff
@@ -79,6 +79,10 @@ export class TaskListCloudComponentPage {
         return this.dataTable.selectRow(column.name, taskName);
     }
 
+    selectRowById(taskName) {
```
Comment (DenysVuika) [inline lib/testing/src/lib/process-services-cloud/pages/task-list-cloud-component.page.ts]: missing types
```suggestion
    selectRowById(taskName: string) {
```

File: lib/testing/src/lib/process-services-cloud/pages/task-list-cloud-component.page.ts [hunk 3]
```diff
@@ -79,6 +79,10 @@ export class TaskListCloudComponentPage {
...
+        return this.dataTable.selectRow(column.id, taskName);
+    }
+
     getRow(taskName) {
```
Comment (DenysVuika) [inline lib/testing/src/lib/process-services-cloud/pages/task-list-cloud-component.page.ts]: missing types
```suggestion
    getRow(taskName: string) {
```

File: lib/testing/src/lib/process-services-cloud/actions/process-definitions.service.ts [hunk 4]
```diff
@@ -44,4 +44,12 @@ export class ProcessDefinitionsService {
             }
         }
     }
+
+    async getProcessDefinitionByName(processDefinitionName, appName) {
```
Comment (marouanbentaleb) [inline lib/testing/src/lib/process-services-cloud/actions/process-definitions.service.ts]: We're missing types here.

Comment (DenysVuika) [reply lib/testing/src/lib/process-services-cloud/actions/process-definitions.service.ts]:
```suggestion
    async getProcessDefinitionByName(processDefinitionName: string, appName: string) {
```

**Comment by mauriziovitale** [comment]
Very old issue, if it is needed feel free to reopen it

**Pull request closed without merging**